first-party/jib-quarkus-extension-maven/src/main/java/com/google/cloud/tools/jib/maven/extension/quarkus/JibQuarkusExtension.java

```diff
@@ -177,6 +182,11 @@ private void readJibConfigurations(MavenProject project) {
     if (jibPlugin != null) {
       Xpp3Dom configurationDom = (Xpp3Dom) jibPlugin.getConfiguration();
       if (configurationDom != null) {
+        Xpp3Dom fastJarEnabled = configurationDom.getChild("fastJarEnabled");
+        if (fastJarEnabled != null) {
+          this.fastJarEnabled = Boolean.valueOf(fastJarEnabled.getValue());
+        }
```

Comment:
As explained above, you don't read extension-specific parameters from the main Jib config.